pods/account/package.json

```diff
@@ -16,6 +16,7 @@
     "_phase:docker-staging": "rushx docker:staging",
     "bundle": "mkdir -p bundle && esbuild src/__start.ts --bundle --define:process.env.MODEL_VERSION=$(node ../../common/scripts/show_version.js) --minify --platform=node > bundle/bundle.js",
     "docker:build": "../../common/scripts/docker_build.sh hardcoreeng/account",
+    "docker:tbuild": "docker build -t hardcoreeng/account . --platform=linux/amd64 && ../../common/scripts/docker_tag_push.sh hardcoreeng/account",
```

Comment:
why do we need to hardcode platform here?

Reply:
Same for all places, it is to perform quickfix build from arm to amd64